src/WooCommerce.php

```diff
@@ -515,13 +518,21 @@ public static function woocommerce_get_item_data($data, $cartItem) {
   public static function woocommerce_display_item_meta($html, $item, $args) {
     $strings = [];
     $product = $item->get_product();
-    $attributes = array_merge(static::getProductData($product), static::getProductAttributes($product));
-    foreach ($attributes as $attribute) {
-      $strings[] = '<strong class="wc-item-meta-label">' . $attribute['name'] . ':</strong> ' . $attribute['value'];
+    $product_data_set = array_merge(static::getProductData($product), static::getProductAttributes($product));
+
+    if ($delivery_time = get_term(wc_get_order_item_meta($item->get_id(), '_deliverytime'), 'product_delivery_times')->name) {
```

Comment:
Isn't the meta key still called `_lieferzeit`?
```suggestion
    if ($delivery_time = get_term(wc_get_order_item_meta($item->get_id(), '_lieferzeit'), 'product_delivery_times')->name) {
```

Reply:
> Isn't the meta key still called _lieferzeit?

Not in this case. '_lieferzeit' is our custom meta field for __product__ delivery times. '_deliverytime' is the meta field for __order items__ delivery time added by woocommerce-german-market.

I will refactor this in a more defensive way, to prevent errors if woocommerce-german-market is not present/disabled.